src/components/form/templates/array-field-collapsible-item.ts

```diff
@@ -56,14 +59,20 @@ export class CollapsibleItemTemplate extends React.Component {
     }
 
     public render() {
-        const { data, schema, index, formSchema } = this.props;
+        const { data, schema, formSchema } = this.props;
+        const inputs = {
```

Comment:
We might want to name this `props`, since that is what it is called in both React and Stencil